app/assets/javascripts/spree/frontend/affirm_checkout.js

```diff
@@ -0,0 +1,74 @@
+(function(){
+  /* only include this setup once */
+  if (!window.AffirmPaymentMethods) {
```

Comment:
Any time we're using a variable to "guard" against doing things twice, we should rethink our approach. There are many utilities and paradigms for accomplishing this in javascript. Our code should be declarative when defining objects, not conditional.